saleor/graphql/payment/mutations/transaction/transaction_request_refund_for_granted_refund.py

```diff
@@ -33,15 +33,20 @@ class TransactionRequestRefundForGrantedRefund(BaseMutation):
     class Arguments:
         id = graphene.ID(
             description=(
-                "The ID of the transaction. One of field id or token is required."
+                "The ID of the transaction. One of field id or token is required, "
+                "if `transactionItem` is not already assigned to the "
+                "`orderGrantedRefund`. If `transactionItem` is already assigned to the "
+                "grantedRefund the field will be ignored."
             ),
             required=False,
         )
         token = UUID(
             description=(
-                "The token of the transaction. One of field id or token is required."
-            )
-            + ADDED_IN_314,
```

Comment:
Is it intended to remove 314 label?

Reply:
Yes. This mutation was introduced in 3.15